src/utils/getClientSdk.ts

```diff
@@ -1,12 +1,18 @@
-import { ClientSdk, CustomClientSdk } from '../models/core/ClientSdk';
+import { AdditionalHttpHeaderValues } from '../models/core/AdditionalHttpHeaderValues';
 import packageJson from '../../package.json';
 
 const { version } = packageJson;
 
-export function getClientSdk(customClientSdk?: CustomClientSdk): ClientSdk {
+export function getClientSdk(
+  additionalHttpHeaderValues?: AdditionalHttpHeaderValues
+): Record<string, string> {
```

Comment:
can the return value type be `ClientSDKHeaderValues`?

Reply:
I don't think that would work because `ClientSDKHeaderValues` specifically prevents the Core header from being specified, which is being added in this function

Reply:
no because we specify ANSWERS_CORE here. I thought about it for a sec and I don't think we need an internal type here since all HttpServiceImpl cares about is that it's a Record<string, string>. I think having an extra type complicates things slightly